pylib/mozhginfo/mozhginfo/push.py

```diff
@@ -0,0 +1,130 @@
+class Push(object):
+    """This represents a Push"""
+
+    def __init__(self, **kwargs):
+        """
+            We set Push object default as dict, and we will
+            give each push a push id when we init it.
+        """
```

Comment:
Could you please improve this docstring?
Add things like :param push_id:
There are examples in mozilla_ci_tools.